Dockerfile

```diff
@@ -15,4 +15,4 @@ USER flowuser
 # Note: EXPOSE is not supported by Heroku, application must respect $PORT
 EXPOSE 5000
 
-CMD ["node", "index.js"]
```

Comment:
Maybe we should use `npm start` here?

Reply:
No I don't want to run NPM in production...